libnestutil/sort.h

```diff
@@ -68,7 +70,7 @@ median3_( const std::vector< T >& vec,
 template < typename T1, typename T2 >
 void
 insertion_sort( std::vector< T1 >& vec_sort,
```

Comment:
why do we need the combination of `vector` and `BlockVector`? shouldn't the sorting always be called on two `BlockVector`s? there's heavy code duplication here, so either this function should be removed or integrated with the sort on two `BlockVector`s